docsite/rst/intro_configuration.rst

```diff
@@ -70,7 +70,7 @@ Actions are pieces of code in ansible that enable things like module execution,
 This is a developer-centric feature that allows low-level extensions around Ansible to be loaded from
 different locations::
 
-   action_plugins = ~/.ansible/plugins/action_plugins/:/usr/share/ansible_plugins/action_plugins
+   action_plugins = ~/.ansible/plugins/action:/usr/share/ansible/plugins/action
```

Comment:
I am in alignment with @bcoca.  The typical naming patterns are to use `x_plugins` and we should not diverge from that in our examples..